Jenkinsfile

```diff
@@ -67,6 +67,27 @@ pipeline {
         }
       }
     }
+
+    stage ('Publish to rubygems.org') {
+      environment {
+        GEM_HOST_API_KEY = credentials('jenkins-rubygems-api-key')
+      }
+      when {
+        anyOf {
+          branch 'master'
+          buildingTag()
+        }
+        steps {
+          container('builder') {
+            script {
+              withCredentials([]) {
```

Comment:
Sure about that one? @llambeau maybe?

Reply:
yeah of course this should not be herre.